test/data/deploy/inference-service-to-delete.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: serving.kserve.io/v1beta1
+kind: InferenceService
+metadata:
+  name: dummy-inference-service
+  namespace: default
+  labels:
+    "mr-inference-service-id": "4"
+  finalizers:
```

Comment:
This finalizer will be added to the isvc created by mr controller? The finalizer of the isvc will not be removed by mr controller, right?

Reply:
mmm not exactly, the finalizer will be added my the odh-model-controller itself (see https://github.com/opendatahub-io/odh-model-controller/pull/135/files#diff-e628c4483237f3dba685d802623c535845221a01b2bd2e3782a82ab9df48320aR120-R129) and then it will be removed by the model controller as well.

The ISVC are going to be created by the dashboard as it is right now